gatorgrade/output/tests/test_function.py

```diff
@@ -0,0 +1,66 @@
+""" Tests to ensure the output_functions.py functions work properly. """
+
+import pytest
+from colorama import init, Fore
```

Comment:
Instead of importing the individual functions/classes that you're planning on using (especially `init`), it would be better to just `import colorama` (if you'd like a shorter name, you can use `import colorama as color` or something similar) and refer to `init` as `colorama.init()` to avoid confusion.

Reply:
Great idea, I changed the import to `import colorama as color` as you suggested, which made the `init` statement `color.init()` which is much more readable. Thanks!